config.yaml

```diff
@@ -33,5 +33,5 @@ params:
   prodFormLink: "https://docs.google.com/forms/d/e/1FAIpQLSev-5clADSdkwi_wiFqBCAECeIoAQDE91chBbeWbvyTjRCeYg/viewform"
   infoMailToLink: "mailto:info@crossplane.io"
   upboundGithubLink: "https://github.com/upbound"
-  latest: 1.9
-  repoLink: "https://github.com/crossplane/crossplane"
\ No newline at end of file
+  latest: 1.10
```

Comment:
This needs to be a string. Found a place I missed with the fix in #165

Reply:
Thanks for catching @plumbis! Updated :)